examples/next-prisma-starter/vitest.config.ts

```diff
@@ -8,5 +8,6 @@ export default defineConfig({
     alias: {
       '~/': fileURLToPath(new URL('./src/', import.meta.url)),
     },
+    setupFiles: ['dotenv/config'],
```

Comment:
Vitest failed **sometimes** based on bad env vars

Not sure how it ever worked